packages/drivers/routerlicious-driver/src/test/restWrapper.spec.ts

```diff
@@ -22,8 +23,16 @@ describe("RouterliciousDriverRestWrapper", () => {
     const token2 = "9876-auth-token-zyxw";
     const token3 = "abc-auth-token-123";
     let tokenQueue: string[] = [];
-    // Pop a token off tokenQueue to generate an auth header
-    const getAuthHeader = async () => `Bearer ${tokenQueue.shift() ?? ""}`;
+
+    const getToken: (refresh?: boolean) => Promise<ITokenResponse | undefined>
```

Comment:
nit: instead of declaring the function type right before declaring the function, instead just declare the function with appropriate types.

```ts
const getToken = async (refresh?: boolean): Promise<ITokenResponse | undefined> => new Promise(...);
```